mods/tuxemon/db/npc/red.json

```diff
@@ -1,4 +1,6 @@
 {
     "slug": "npc_red",
-    "sprite_name": "red"
+    "sprite_name": "red",
+    "combat_front": "red_front.png",
+    "combat_back": "red_back.png"
```

Comment:
this is good.  please go through the other npcs and add front/back sprites.  i think if there is no good default, you should use red.

Reply:
For all NPCs, I have added either red or maple front/back sprites.
Knight and professor both have their own front sprites so those are used there.